content/2016.01.loadtesting.with-oauth-part-2.rst

```diff
@@ -0,0 +1,264 @@
+How to load test a OAuth-protected HTTP API? (Part 2)
+#####################################################
+
+:lang: en
+:date: 2016-01-05
+:url: load-testing-a-http-api-which-uses-oauth-for-authentication-part-2
+:summary: This is part 2 of a series on loadtesting oauth APIs explaining how to test the performance of an HTTP API when its access depends on an external OAuth service.
+
+This is part 2 of a series on loadtesting oauth APIs.
+
+Introduction
+============
+
+`In the previous article </en/load-testing-a-http-api-which-uses-oauth-for-authentication>`_
+we have seen that:
+
+- We would create the OAuth Bearer Token before the load-test start.
+- We can use environment variables to use the token in the load-test.
+
+In this article we will have a closer look at how we can generate
+tokens for Firefox Account and pass them to the load-test.
+
+
+PyFxA: the Python Firefox Account client
+========================================
+
+`PyFxA <https://github.com/mozilla/PyFxA>`_ is a Python library that
+aims providing helpers to interact with the Firefox Account ecosystem.
+
+
+Using PyFxA with httpie
+-----------------------
+
+A while ago we added support for 
+`Requests Authentication <http://docs.python-requests.org/en/latest/user/authentication/>`_
```

Comment:
Consider reversing the phrasing: "To ease the use of FxA bearer tokens, we've created a authentication policy for the requests library, which can also be used with the httpie client".

I would specify directly a link to this authentication policy.